File: core/src/main/java/uk/lobsterdoodle/namepicker/overview/OverviewCardActionsCallback.java
package uk.lobsterdoodle.namepicker.overview;

public interface OverviewCardActionsCallback {
    void launchEditGroupNamesScreen(long groupId);
    void launchEditGroupDetailsScreen(long groupId);
    void deleteGroup(long groupId);
    void launchSelectionScreen(long groupId);
}

File: core/src/main/kotlin/uk/lobsterdoodle/namepicker/overview/OverviewCardActionsCallback.kt
package uk.lobsterdoodle.namepicker.overview

interface OverviewCardActionsCallback {
    fun launchEditGroupNamesScreen(groupId: Long)
    fun launchEditGroupDetailsScreen(groupId: Long)
    fun deleteGroup(groupId: Long)
    fun launchSelectionScreen(groupId: Long)
}
